main.py

```diff
@@ -261,7 +262,7 @@ def get_cover_art(filename: str) -> Optional[File]:
         audio = MutagenFile(filename)
 
         # In each case, ensure audio tags are not None or empty
-        if isinstance(audio, ID3FileType):
+        if isinstance(audio, ID3FileType) or isinstance(audio, WAVE):
```

Comment:
```suggestion
        # mutagen.wave.WAVE is not an ID3FileType, though its tags are
        # of type mutagen.id3.ID3
        if isinstance(audio, ID3FileType) or isinstance(audio, WAVE):
```

just to explain why we're doing this.